WORKSPACE

```diff
@@ -23,13 +23,33 @@ http_file(
 )
 
 maven_jar(
-  name = "grpc_all_artifact",
-  artifact = "io.grpc:grpc-all:0.13.2",
+  name = "grpc_benchmarks_artifact",
+  artifact = "io.grpc:grpc-benchmarks:1.0.0",
+)
+
```

Comment:
The travis build seems to think that there is still a missing artifact here.

1) clientStreamRpcMetrics(me.dinowernli.grpc.prometheus.integration.MonitoringClientInterceptorIntegrationTest)
java.lang.NoClassDefFoundError: io/grpc/protobuf/lite/ProtoLiteUtils